components/auth/SignOut.tsx

```diff
@@ -2,11 +2,17 @@ import type { ButtonProps } from "react-bootstrap"
 import { Button } from "../bootstrap"
 import { auth } from "../firebase"
 
+// Signout, and redirect to the home page.
+const handleSignout = () => {
+  auth.signOut()
+  location.assign(`/`) // Redirect to the home page.
```

Comment:
Let's use the nextjs router for this:

```
import Router from 'next/router'

Router.push('/')
```

Reply:
I have updated this on the latest commit.
Thank you